docs/maintain/maintain-guides-how-to-upgrade.md

```diff
@@ -6,72 +6,68 @@ descriptions: Steps on how to upgrade your validator node.
 ---
 
-Validators perform critical functions for the network, and as such, have strict uptime requirements.
-Validators may have to go offline for periods of time to upgrade the client software or the host
-machine. Usually, standard upgrades will only require you to stop your server, replace the binary
-and spin it up again. Validators may also need to perform maintenance, such as resizing a disk,
-where you cannot continue running the node on the current server. This guide will walk you through
-upgrading your machine and keeping your validator online.
+Validators perform critical functions for the network, and as such, have strict uptime requirements. Validators may have to go offline for short-periods of time to upgrade client software or to upgrade the host machine. Usually, standard client upgrades will only require you to stop the service, replace the binary and restart the service.  This operation can be executed within a session and if performed correctly will not produce a slashable event.
 
-The process can take several hours, so make sure you understand the instructions first and plan
-accordingly.
+Validators may also need to perform long-lead maintenance tasks that will span more than one session.  Under these circumstances an active validator may chill his stash and be removed from the active set of validators.  Alternatively, the validator may substitue the active validator server with another allowing the former to undergo maintainance activities.
+
+This guide will provide an option for validators to seamlessly substitute an active validator server to allow for maintenance operations. 
+
+The process can take several hours, so make sure you understand the instructions first and plan accordingly.
 
 > Note: keep an eye out on new releases from the community and upgrade/downgrade accordingly.
 
 ## Key Components
 
 ### Session Keys
 
-Session keys are stored in the client and used to sign validator operations. They are what link your
-validator node to your Controller account. You cannot change them mid-Session.
+Session keys are stored in the client and used to sign validator operations. They are what link your validator node to your Controller account. If you change them within a session you will have to wait for the current session to finish and a further two sessions to elapse before they are applied.
 
 [More info about keys in Polkadot.](../learn/learn-keys.md)
 
-### Database
+### Keystore
 
-Validators keep a database with all of their votes. If two machines have the same Session keys but
-different databases, they risk equivocating. For this reason, we will generate new Session keys each
-time we change machines.
+Each validator server contains essential private keys in a folder called the keystore. These keys are used by a validator to sign transactions at the network level.  If two or more validators sign certain transactions using the same keys it can lead to an equivocation slash.
```

Comment:
```suggestion
Each validator server contains essential private keys in a folder called the *keystore*. These keys are used by a validator to sign transactions at the network level.  If two or more validators sign certain transactions using the same keys, it can lead to an [equivocation slash](../learn/learn-staking.md/#slashing).
```